src/components/MapFacetContents.js

```diff
@@ -31,19 +31,25 @@ type MapFacetContentsProps = {
   size: any;
   /** The public key with which to connect to the mapbox public apis. */
   mapboxKey: string;
-  /** Custom location pointer glyphicon, default is the map-marker glyphicon */
-  glyph: string | null;
+  /** Custom location pointer pointerGlyphicon, default is the map-marker glyphicon */
+  pointerGlyph: string | null;
   /** Custom location pointer image uri, if available image is shown instead of glyphicon */
-  uri: string | null;
-  /** Custom tooltip text to convey what the location pointers are for, for e.g. customers, transaction location etc */
+  pointerImageUri: string | null;
+  /**
+   * Custom tooltip text to convey what the location pointers are for, for e.g. customers, transactions etc
+   * It should specify the singular and plural forms to be used
+   * For example a tooltip for cats would be '{} cat|{} cats'
+   * If the string is not in the format specified above,
+   * it would be used as it is concatenated with the count.
```

Comment:
I might just change "concatenated with" to be "appended to" so it's unambiguous that the order will be first the count and then the tooltip string.

Reply:
Changed!